starter-code/css/main.css

```diff
@@ -1,8 +1,134 @@
-/*
-Colors:
+body {
+  padding: 0;
+  margin: 0 auto;
+  width: 100%;
+}
 
-Text: 1A1A1A
-Green: #00B172
-White: #FFF
+a {
+  text-decoration: none;
+  color: black;
+}
 
-*/
+li a {
+  padding-right: 30px;
+}
+
+.wrapper {
+  font-family: 'Open Sans', sans-serif;
+  line-height: 1.4;
+  font-size: 18px;
+  color: #444;
+  background-color: #fff;
+}
+
+nav {
+  padding: 0;
+  margin: 0 20px 0 0;
+  text-align: center;
+}
+
+.navWrap {
+  width: 89%;
+  float: right;
+}
+
+.topLeft {
+  display: inline-block;
+}
+
+a > img {
+  width: 145px;
+}
+
+a span {
+  background-image: url("../images/spotify-logo.png");
+  margin-top: 20px;
+  margin-bottom: 20px;
+  width: 132px;
+  height: 40px;
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  float: left;
+}
+
+.topLinks {
+  padding-left: 63%;
+  padding-top: 1%;
+  margin-bottom: 20px;
+  margin-top: 20px;
+  line-height: 19px;
+}
+
+.topLinks > li {
+  display: inline-block;
+}
+
+
+.firstHalf {
+  background-image: url(../images/landing.jpg);
+  background-size: cover;
+  background-position: center;
+  height: 600px;
+}
+
+.firstHalf > h1 {
+  color: #fff;
+  text-align: center;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 48px;
+  padding-top: 17%;
+  margin: 0;
+}
+
+.firstHalf > p {
+  color: #fff;
+  text-align: center;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 20px;
+  font-weight: 100;
+  width: 550px;
+  margin: 0 auto;
+}
+
+.bottomHeader {
+  text-align: center;
+  font-size: 27px;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 100;
+}
+
+.secondHalf {
+  display: inline-block;
+
+}
+
+.bottomHeader > h2 {
+  border-bottom: 4px solid #1DB954;
+  text-align: center;
+  display: inline-block;
+}
+
+.bottomImages {
```

Comment:
try using the box-sizing: border-box; property to get the format right on this one. Border box allows us set margin and padding and have it take away from a set width rather then add a set width. 

`
.bottom-images {
    width: 33%;
    box-sizing: border-box;
    float: left;
    padding: 20px 40px 20px;
    text-align: center;
}

This will give you a clean layout close to the original

`